helix-core/src/comment.rs

```diff
@@ -1,17 +1,18 @@
 use crate::{
     find_first_non_whitespace_char, Change, Rope, RopeSlice, Selection, Tendril, Transaction,
 };
-use core::ops::Range;
 use std::borrow::Cow;
 
 fn find_line_comment(
```

Comment:
Can you please add a comment `///` for this? The return is getting a bit long so we may want to explain what it returns.

Reply:
You want me to add a documentation comment for `find_line_comment`?

Reply:
Ok